docs/competition/challenges.rst

```diff
@@ -16,7 +16,7 @@ There are 8 possible :term:`agility challenges<Agility Challenge>`. A descriptio
 Faulty Parts Challenge
 ----------------------
 
-Faulty parts are parts that are not suitable for use in the competition. If an order is submitted with faulty parts, these parts are not considered for scoring. Faulty parts are identified by quality control sensors, which are attached to AGVs. This challenge will only occur for kitting orders. 
+Faulty parts are parts that are not suitable for use in the competition. If an order is submitted with faulty parts, the score for that order will be zero. Faulty parts are identified by quality control sensors, which are attached to AGVs. This challenge will only occur for kitting orders. 
```

Comment:
The score for the quadrant will be zero, not the entire order.

Reply:
I fixed this but I don't know how to indicate that it is fixed?